src/components/CohortListItem/CohortListItem.js

```diff
@@ -4,6 +4,8 @@ import { ListItem } from './CohortListItemStyle';
 import { Badge } from '../../GlobalStyle';
 
 const CohortListItem = ({ cohortID, key }) => {
+  const formatedLabel =
```

Comment:
Not a huge deal but I think there's a type here. It probably should be `formattedLabel` 🙂